### PR TITLE
Batch quoter calls with multicall

### PR DESCRIPTION
**Pull request by Max-3-7**

### CODE REVIEW

**Review by gaepsuni — APPROVED** [review]
lgtm! small nit, if possible we might want to add a test case

**Comment by Max-3-7** [comment]
sorry I didn't see your message before merging, we already have tests to cover `getTradesExactOut` and `getTradesExactIn` (they do not pass if you make the multicall fail for example) so it seems good enough?
unless you had something specific in mind?

**Comment by gaepsuni** [comment]
> sorry I didn't see your message before merging, we already have tests to cover `getTradesExactOut` and `getTradesExactIn` (they do not pass if you make the multicall fail for example) so it seems good enough? unless you had something specific in mind?

Oh I see then it should be good enough!

**Pull request merged**
